neuralop/layers/complex_utils.py

```diff
@@ -0,0 +1,49 @@
+"""
+Functionality for handling complex-valued spatial data
+"""
+
+from copy import deepcopy
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+
+def CGELU(x: torch.Tensor):
+    """Complex GELU activation function
+    Follows the formulation of CReLU from Deep Complex Networks (https://openreview.net/pdf?id=H1T2hmZAb)
+    Args:
+        x: tensor
+    """
+
+    return F.gelu(x.real).type(torch.cfloat) + 1j * F.gelu(x.imag).type(
+        torch.cfloat
+    )
+
+
+def ctanh(x: torch.Tensor):
+    """Complex-valued tanh stabilizer"""
+    return torch.tanh(x.real).type(torch.cfloat) + 1j * torch.tanh(x.imag).type(
+        torch.cfloat()
+    )
+
+
+def apply_complex(fr, fi, x, dtype=torch.cfloat):
```

Comment:
Could we make the variable names clearer? e.g. expand `f`, use real and imag etc.